source/adios2/toolkit/format/bp/BPBase.cpp

```diff
@@ -368,6 +367,10 @@ BPBase::GetTransportIDs(const std::vector<std::string> &transportsTypes) const
         {
             id = METHOD_POSIX;
         }
+        else if (method == "File_FlexNVMe")
+        {
+            id = METHOD_FLEXNVME;
+        }
```

Comment:
Let's revert all of this file and bpbase.h. We haven't tested it so i don't think it makes any difference.

Reply:
👍